front/src/containers/TicketOrder/orderSection.scss

```diff
@@ -86,8 +81,15 @@
         stroke-width: 10;
         transition: 0.5s;
 
+        &.loveseat{
+            stroke: rgb(151, 71, 104);
```

Comment:
You'd better define some variables with colors and reuse them across the application. This will greatly simplify your life in case you need to adjust them.